src/recipes/thought.ts

```diff
@@ -0,0 +1,48 @@
+import { defineRecipe } from '@pandacss/dev'
+
+const thoughtRecipe = defineRecipe({
+  className: 'thought',
+  base: {
+    maxWidth: '100%',
+    /* do not set font-weight or it will override =heading style. */
+    marginTop: '0',
+    display: 'inline-block',
+    verticalAlign: 'top',
+    whiteSpace: 'pre-wrap',
+    '@media (max-width: 500px)': {
+      marginTop: { _android: '-2.1px' },
+      marginLeft: { _android: '0.5em' },
+    },
+    '@media (min-width: 560px) and (max-width: 1024px)': {
+      marginTop: { _android: '-0.1px' },
+      marginLeft: { _android: '0.5em' },
+    },
+  },
+  variants: {
+    /** Assign annotation height on single line truncated url. */
+    singleline: {
```

Comment:
I know this was the existing name, but can you rename `singleline` to `ellipsizedUrl`? 

I just realized that calling it `singleline` makes it seem like it is the opposite of `multiline`, but it's really a different concept.

Reply:
Yes, of course.